### PR TITLE
ZENKO-3600 Bump backbeat 8.2.12 for Zenko 1.x

### PR DESCRIPTION
**Pull request by nicolas2bert**

### CODE REVIEW

**Review by miniscruff — APPROVED** [review]

**Review by naren-rajendran — APPROVED** [review]

File: eve/workers/end2end/requirements.sh [hunk 1]
```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
 
 export HELM_VER=2.16.7
+export HELM3_VER=3.5.4
```
Comment (tcarmet) [inline eve/workers/end2end/requirements.sh]: Is there anything stopping you from fully removing helm2 ?

Comment (nicolas2bert) [reply eve/workers/end2end/requirements.sh]: Do you mind moving your comment here? https://github.com/scality/Zenko/pull/1289
I just rebased from this branch. Thank you.

**Comment by nicolas2bert** [comment]
@bert-e reset

**Review by naren-rajendran — APPROVED** [review]

**Comment by naren-rajendran** [comment]
@bert-e reset

**Comment by nicolas2bert** [comment]
@bert-e reset

**Pull request merged**